packages/ui/src/council/hooks/useCouncilStatistics.ts

```diff
@@ -0,0 +1,19 @@
+import { useApi } from '@/common/hooks/useApi'
+import { useCouncilStatisticsQuery } from '@/council/queries'
+
+export const useCouncilStatistics = () => {
+  const { api } = useApi()
+  const { loading, data } = useCouncilStatisticsQuery()
+
+  return {
+    isLoading: loading,
+    budget: {
+      amount: data?.budgetSetEvents[0]?.newBudget,
+      PERIOD_LENGTHS: api?.consts.council.budgetRefillPeriod,
+    },
+    reward: {
+      amount: data?.councilorRewardUpdatedEvents[0]?.rewardAmount,
```

Comment:
As for the values – the `councilorReward()` returns the reward per block, but IIRC the reward is paid in periods (hence, the reward period). A similar situation was with Working Group payouts.

I other words, I think that we should display `councilorRewars * electedMemberRewardPeriod` as the `reward.amount`.

Reply:
See the docs: https://github.com/Joystream/pioneer/blob/cc406bf8721aeebd60d7bcac0e2ffeecee947bd5/packages/types/augment/augment-api-query.ts#L265-L268